projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -61,7 +66,7 @@ public Result<Project, ApplicationException> createProject(String title, String
 
   private ProjectCode createRandomCode() {
     ProjectCode code = ProjectCode.random();
-    while (!projectRepository.find(code).isEmpty()) {
+    while (!projectRepository.find(code).isEmpty() || projectDataRepository.projectExists(code)) {
```

Comment:
Do we want to only check openbis? Should they be out of sync? Can there be projects that are in the project repostiory but not stored in openbis? 

I have the feeling only one condition is enough

Reply:
Unless we change project creation in the openbis connector this will lead to an error, if a project exists there. They should not be out of sync, but I think it's important to check. Maybe also concerning old projects?

Reply:
hm ok, I actually thought we added the check in the project repository only because we did not have the openbis implementation at that time.

But we also dont lose anything to keep it like this. Wont object.